Desktop, Cli: enex_to_md: support italic in span tags

JOJ0 opened the pull request:
Hi, this should theoretically be a no-brainer. It is just an addition to the "span tags format conversion feature" you merged a couple of weeks ago (#1708). I also added a html and md pair test and it passes. Additionally I did a lot of testing with my old Evernote notes.

Anyway, I am experiencing an odd issue: Some tests don't pass and I think I didn't break them: 

```
Error converting file: mathjax_block.html
Error converting file: mathjax_inline.html
Error converting file: table_with_pipe.html
```

By any chance, are they currently broken or should they work?

Thanks

laurent22 commented on ReactNativeClient/lib/import-enex-md-gen.js:
```diff
@@ -395,7 +395,15 @@ function isSpanStyleBold(attributes) {
+
+function isSpanStyleItalic(attributes) {
+	let style = attributes.style;
+	if (style.includes('font-style: italic;')) {
```
That seems a bit fragile - what if it is set as `font-style:italic ;`? Maybe spaces should be removed and the string made lowercase before testing it?

laurent22 commented on ReactNativeClient/lib/import-enex-md-gen.js:
```diff
@@ -395,7 +395,15 @@ function isSpanStyleBold(attributes) {
+	} else {
 		return false;
 	}
 }
```
Can be written as `return style.includes('...')`

laurent22 commented:
Thanks for the PR, overall it looks good, and thanks also for adding the tests.

If some tests are broken they'll need to be fixed, unless they were already broken before your changes. Could you confirm please? Another issue I guess is that the CI is not detecting that any test is failing so that needs to be fixed too, but not something you need to worry about.

JOJ0 commented:
Hi
* incorporated all your suggestions and amended them to first commit
* fixed a conflict in CI (because of a now unnecessary debug message)

The thing with tests not running through were my fault: I forgot to rebuild the CliClient and tests! All tests passing now.

laurent22 commented:
Looks good now, thanks for the PR @JOJ0!
